code/game/objects/items/blueprints.dm

```diff
@@ -228,8 +228,8 @@
 		FD.CalculateAffectingAreas()
 
 	interact()
-	message_admins("A new room was made by [ADMIN_LOOKUPFLW(usr)] at [ADMIN_VERBOSEJMP(src)] with the name [str]")
-	log_game("A new room was made by [key_name(usr)] at [AREACOORD(src)] with the name [str]")
+	message_admins("A new room was made by [key_name_admin(usr)] at [ADMIN_VERBOSEJMP(usr)] with the name [str]")
+	log_game("A new room was made by [key_name_admin(usr)] at [AREACOORD(usr)] with the name [str]")
```

Comment:
```suggestion
	log_game("A new room was made by [key_name(usr)] at [AREACOORD(usr)] with the name [str]")
```
just `key_name` here will do since this will be logged to a file and thus should not have hrefs

Reply:
At first I wasn't sure wheter logs are for outside game use or for in game viewing, I assumed latter, but it seems I was wrong.

Reply:
Understandable.
The ingame log viewer currently uses the following proc for that:
`usr.create_log(LOG_TYPE, message, target, where)` where ` target` and `where` are optional parameters. Where is auto filled if left empty